src/pages/Designer/Designer.js

```diff
@@ -117,55 +117,111 @@ const Designer = () => (
           <BarChart size="large" color="TabularGreen" />
         </Box>
       </Box>
+      <ResponsiveContext.Consumer>
+        {(responsive) =>
+          responsive === 'large' || responsive === 'xlarge' ? (
+            <Stack>
+              <Box
+                background="YellowBackground"
+                height="600px"
```

Comment:
hard coding this box will be tricky on responsive behavior when the tiles need to be set in a column.